zscript/gearbox/event_handler.zs

```diff
@@ -315,6 +318,29 @@ class gb_EventHandler : EventHandler
     mActivity.openInventory();
   }
 
+  private ui
+  void selectNextWeapon(bool withTickSound = true)
+  {
+    bool success = mWeaponMenu.selectNextWeapon();
+    if (success && withTickSound) mSounds.playTick();
+  }
+
+  private ui
+  void selectPrevWeapon(bool withTickSound = true)
+  {
+    bool success = mWeaponMenu.selectPrevWeapon();
+    if (success && withTickSound) mSounds.playTick();
+  }
+
+  private ui
+  bool selectWeaponSlot(int slot, bool withTickSound = true)
+  {
+    bool success = mWeaponMenu.selectSlot(slot);
+    if (success && withTickSound) mSounds.playTick();
+
+    return success;
+  }
```

Comment:
Proposal: instead of these functions, make
```
private ui
void tickIf(bool mustTick)
{
  if (mustTick) mSounds.playTick();
}
```
and wrap mWeaponMenu calls where needed (where withTickSound is true). I believe the code will be more self-documenting this way. What do you think?

Reply:
That's a nice idea, it's a lot cleaner and more explicit, and less private methods.